pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcTitle.scala

```diff
@@ -0,0 +1,84 @@
+package weco.pipeline.transformer.marc_common.transformers
+import grizzled.slf4j.Logging
+import weco.pipeline.transformer.exceptions.ShouldNotTransformException
+import weco.pipeline.transformer.marc_common.models.{MarcRecord, MarcSubfield}
+
+object MarcTitle extends MarcDataTransformer with Logging {
+
+  type Output = Option[String]
+
+  // Populate wwork:title.  The rules are as follows:
```

Comment:
Nit:
```suggestion
  // Populate work:title.  The rules are as follows:
```

Reply:
Apparently this means Wellcome Work, from the days of the original ontology development.  However, I can't find that written down anywhere, and I'm not sure that it's at all pertinent.  However, it is ubiquitous (https://github.com/search?q=org%3Awellcomecollection%20wwork&type=code)